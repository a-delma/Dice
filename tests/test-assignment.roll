
Int a;
Bool b;
b = false;
a = 97;

putChar(a);
