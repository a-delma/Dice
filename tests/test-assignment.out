
Int a;
a = 72;

putChar(a);
